bin/citrea/tests/test_client/mod.rs

```diff
@@ -482,6 +482,18 @@ impl TestClient {
         eth_logs
     }
 
+    pub(crate) async fn eth_get_proof(
+        &self,
+        address: Address,
+        keys: Vec<U256>,
+        block_number: Option<BlockNumberOrTag>,
+    ) -> Result<EIP1186AccountProofResponse, Box<dyn std::error::Error>> {
+        self.http_client
```

Comment:
let's use alloy's client here to see if it can parse our proof

Reply:
they don't support block id param. But we use their EIP1186AccountProofResponse so we are good